services/consumer/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"crypto/tls"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"k8s.io/klog"
+
+	pb "github.com/red-hat-storage/ocs-operator/services/provider/pb"
+)
+
+var (
+	storageProviderEndpoint = flag.String("endpoint", "localhost:50051", "The storageProviderEndpoint")
+	storageConsumerUID      = flag.String("storageConsumerUID", "UID", "The UID of storageConsumer CR")
+)
+
+func main() {
+	flag.Parse()
+
+	config := &tls.Config{
+		InsecureSkipVerify: true,
+	}
+
+	opts := []grpc.DialOption{
+		grpc.WithTransportCredentials(credentials.NewTLS(config)),
+		grpc.WithBlock(),
+	}
+
+	conn, err := grpc.Dial(*storageProviderEndpoint, opts...)
+	if err != nil {
+		klog.Fatalf("failed to dial: %v", err)
+	}
+	defer conn.Close()
+
+	c := pb.NewOCSProviderClient(conn)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	defer cancel()
```

Comment:
just use `providerclient.NewProviderClient` ?

Reply:
Why? The name of gRPC service is OCSProvider so client name will be NewOCSProviderClient, right?

Reply:
@bindrad I think the ask here is to use the method on the providerclient struct instead of accessing the protobuf struct directly as the client is supposed to be an obstruction of the protobuf struct

Reply:
Understood, updated in latest commit